spec/fizzbuzz_spec.rb

```diff
@@ -4,19 +4,33 @@
 describe 'SuperFizzBuzz' do
   let(:script) { SuperFizzBuzz.new }
 
-  it "returns 'Fizz' when my input is divisible by 3" do
-    expect(script.run(3)).to eq "Fizz"
-  end
-  
-  it "returns 'Buzz' when my input is divisible by 5" do
-    #implement your test here
-  end
+  context 'fizz buzz test' do
+    fizzy_numbers = [3,6,9,12,18,21,24,27,33,36,39,42,48,51,54,57,63,66,69,72,78,81,84,87,93,96,99]
+    fizzy_numbers.each do |number|
```

Comment:
I think right now this is going to generate a whole bunch of tests along the lines of, "fizz buzz test returns 'Fizz' when my 3 is divisible by 3", "fizz buzz test returns 'Fizz' when my 6 is divisible by 3", etc. I think it would be much nicer if this was instead a single test covering all of your test numbers.